CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # The Taichi Programming Language
 #*********************************************************************
 
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Do we need this?

Reply:
yep, [target_link_options](https://cmake.org/cmake/help/latest/command/target_link_options.html) is not available before 3.13. Given 3.13 was more than 3 years old (released in 2019), it is ok to be minimum required ;/